orchestra/static/orchestra/project/js/controllers.js

```diff
@@ -0,0 +1,870 @@
+(function () {
+  // 'use strict';
+
+  angular
+    .module('orchestra.project.controllers')
+    .controller('ProjectController', ProjectController);
+
+  ProjectController.$inject = ['$location', '$scope', '$routeParams', '$http', '$sce',
+                            '$compile', '$modal', '$timeout', 'dataService', 'orchestraApi'];
```

Comment:
is angular style that services are prepended with $, and if so, should you prepend the service and orchestra api?